### PR TITLE
UX: Events UI Edits

### PR DESCRIPTION
**Pull request by jordanvidrine**
This PR restyles the current events post UI:

- Minimize size of invitee avatars
- move CTA buttons to bottom
- remove invitee header & show more button (show more is in ellipses drop down)
- restyle border
- restyle month & date size & spacing
- increase font size for title
- add "0 going" placeholder to prevent jumpiness when invitees are rendered

Related PR -> https://github.com/discourse/discourse-calendar/pull/654

**Before**
image

**After**
image

### CODE REVIEW

File: assets/stylesheets/common/discourse-post-event.scss [hunk 1]
```diff
@@ -28,14 +27,16 @@ $interested: #fb985d;
...
-    border: 5px solid var(--primary-low);
+    box-shadow: 0px 0px 0px 3px var(--primary-100);
+    border: 1px solid var(--primary-300);
     display: flex;
     background: var(--secondary);
     margin: 5px 0;
     flex-direction: column;
     flex: 1 0 auto;
-    max-width: 100%;
+    max-width: calc(100% - 2px - 6px);
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:37]: what are these magic numbers?

File: assets/javascripts/discourse/components/discourse-post-event/invitees.gjs [hunk 2]
```diff
@@ -70,11 +71,22 @@ export default class DiscoursePostEventInvitees extends Component {
...
+                <Invitee @invitee={{invitee}} />
+              {{/each}}
+            </ul>
+          </div>
+        </section>
+      {{else}}
+        <section class="event__section no-rsvp">
+          <div class="header">
+            <div class="event-invitees-status">
+              <p class="no-rsvp-description">0 going</p>
```
Comment (jjaffeux) [inline assets/javascripts/discourse/components/discourse-post-event/invitees.gjs:87]: we need to translate this

File: assets/stylesheets/common/discourse-post-event.scss [hunk 3]
```diff
@@ -308,32 +363,56 @@ $interested: #fb985d;
...
+  .no-rsvp-description {
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    letter-spacing: 0.00625rem;
+    font-weight: 400;
+    border-bottom: none;
+    margin: 0;
+  }
+  .event__section.no-rsvp {
+    height: calc(48px - 1em);
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:404]: what is the 1em here?

File: assets/stylesheets/common/discourse-post-event.scss [hunk 4]
```diff
@@ -308,32 +363,56 @@ $interested: #fb985d;
...
+
+.discourse-post-event {
+  .cooked-date,
+  .participants,
+  .event-url .url,
+  .no-rsvp-description {
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    letter-spacing: 0.00625rem;
+    font-weight: 400;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:399]: It feels weird to have a plugin having a different way to render text from core

File: assets/stylesheets/common/discourse-post-event.scss [hunk 5]
```diff
@@ -308,32 +363,56 @@ $interested: #fb985d;
...
+    align-items: center;
+  }
+  .event-invitees-status {
+    margin-left: 4em;
+    .no-rsvp-description {
+      color: var(--primary-medium);
+      font-size: 0.75rem;
+      line-height: 1rem;
+      letter-spacing: 0.0125rem;
+      font-weight: 400;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:415]: same remark

File: assets/stylesheets/common/discourse-post-event.scss [hunk 6]
```diff
@@ -205,7 +246,7 @@ $interested: #fb985d;
     flex-direction: column;
 
     .header {
-      display: flex;
+      display: none;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:249]: why do we have it if we don't show it?

File: assets/stylesheets/common/discourse-post-event.scss [hunk 7]
```diff
@@ -127,21 +140,38 @@ $interested: #fb985d;
...
     .name {
-      font-weight: 700;
       @include ellipsis;
       max-width: 45vw;
+      font-size: 1.25rem;
+      line-height: 1.5rem;
+      letter-spacing: 0;
+      font-weight: 400;
+      white-space: initial;
+      margin-bottom: 0.25rem;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:150]: same question about not using core font sizes

File: assets/stylesheets/common/discourse-post-event.scss [hunk 8]
```diff
@@ -102,21 +103,33 @@ $interested: #fb985d;
...
-    margin-right: 1em;
 
     .month {
       text-align: center;
       color: red;
-      font-size: var(--font-down-1);
+      font-size: 0.6875rem;
+      line-height: 0.75rem;
+      letter-spacing: 0.0125rem;
+      font-weight: 400;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:123]: core font size

File: assets/stylesheets/common/discourse-post-event.scss [hunk 9]
```diff
@@ -102,21 +103,33 @@ $interested: #fb985d;
...
     }
 
     .day {
       text-align: center;
-      font-weight: 500;
-      font-size: var(--font-up-2);
+      font-size: 1.25rem;
+      font-weight: 400;
+      line-height: 1.5rem;
+      letter-spacing: 0.2px;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:132]: core font size

File: assets/stylesheets/common/discourse-post-event.scss [hunk 10]
```diff
@@ -1,15 +1,14 @@
 $interested: #fb985d;
+$show-interested: inherit;
 
 .discourse-post-event {
   display: flex;
+  max-width: 35rem;
```
Comment (jjaffeux) [inline assets/stylesheets/common/discourse-post-event.scss:6]: what is 35 rem?

File: assets/stylesheets/common/discourse-post-event.scss (hunk 1 above)
Comment (jordanvidrine) [reply assets/stylesheets/common/discourse-post-event.scss:37]: The numbers are the 1px border on each side (1+1 = 2px) and the box shadow on each side (3 + 3 = 6px). This way, the total width of the box with border & box shadow always fit inside its container.

File: assets/javascripts/discourse/components/discourse-post-event/invitees.gjs (hunk 2 above)
Comment (jordanvidrine) [reply assets/javascripts/discourse/components/discourse-post-event/invitees.gjs:87]: Oooh. Good find!

File: assets/stylesheets/common/discourse-post-event.scss (hunk 3 above)
Comment (jordanvidrine) [reply assets/stylesheets/common/discourse-post-event.scss:404]: This is the 0.5em padding on top and bottom. So the height is taking into consideration the padding that will be added.

File: assets/stylesheets/common/discourse-post-event.scss (hunk 6 above)
Comment (jordanvidrine) [reply assets/stylesheets/common/discourse-post-event.scss:249]: I think I felt it was too drastic to remove for now? This is a hard one. If someone still wants this header with the "show all" button, I was afraid of removing it.

I am fine to remove it though as we are still giving people the option to show header in the ellipsis menu.

File: assets/stylesheets/common/discourse-post-event.scss (hunk 10 above)
Comment (jordanvidrine) [reply assets/stylesheets/common/discourse-post-event.scss:6]: Its just to decrease the width of the container. I saw it used elsewhere and thought it was a nice width to stick to.

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**